website/source/docs/builders/scaleway.html.md

```diff
@@ -49,7 +49,8 @@ builder.
 -   `image` (string) - The UUID of the base image to use. This is the image
     that will be used to launch a new server and provision it. See
     <https://api-marketplace.scaleway.com/images> get the complete list of the
-    accepted image UUID.
+    accepted image UUID. You can also specify a size compatible with the
+    instance type which will then creae a blank base image.
```

Comment:
Can we get some documentation on examples of valid sizes?

Reply:
I wonder if this would be clearer as an entirely different template option, rather than overloading "image" For example, "blank_image_size", and then we can validate that you can't have both "image" and "blank_image_size" set.